src/ripple/app/tx/impl/SetHook.cpp

```diff
@@ -1222,6 +1262,13 @@ SetHook::setHook()
         std::optional<uint256> newHookOn;
         std::optional<uint256> defHookOn;
 
+        std::optional<uint256> oldOutgoingHookOn;
```

Comment:
```c++
std::optional<uint256> oldHookOn;
...
std::optional<uint256> oldOutgoingHookOn;
std::optional<uint256> oldIncomingHookOn;
```

These never seem to be actually accessed

Reply:
Would definitely make the code a bit easier to read if they were

Reply:
removed